TomeOfKnowledge.playground/Pages/Exploring Expressions.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,50 @@
+//: ## Exploring Expressions
+//: 
+//: Our expressions are more robust than what is needed for typical tabletop play. In fact, our generic expression system is capable of modeling expressions that would otherwise be considered too cumbersome or complex to physically roll. In order to help developers understand what their expressions actually represent, we have added various tools to make working with expressions as easy as possible.
+
```

Comment:
We have the comments sit on top of the expressions in the playground. So this would be removed.